install/kubernetes/helm/istio/values-istio-one-namespace-auth.yaml

```diff
@@ -15,6 +15,4 @@ global:
   # Default is 10s second
   refreshInterval: 1s
 
-istiotesting:
-  oneNameSpace: true
-
+  oneNamespace: true
```

Comment:
could we leave some comment for this?  before, it is a testing param, I can safely ignore it.

Reply:
Added some comments for the `oneNamespace ` parameter.